commits.go

```diff
@@ -164,15 +164,18 @@ type CommitComment struct {
 	Path     string `json:"path"`
 	Line     int    `json:"line"`
 	LineType string `json:"line_type"`
-	Author   struct {
-		ID        int       `json:"id"`
-		Username  string    `json:"username"`
-		Email     string    `json:"email"`
-		Name      string    `json:"name"`
-		State     string    `json:"state"`
-		Blocked   bool      `json:"blocked"`
-		CreatedAt time.Time `json:"created_at"`
-	} `json:"author"`
+	Author   Author `json:"author"`
+}
+
+
+type Author struct {
+	ID        *int       `json:"id"`
+	Username  *string    `json:"username"`
+	Email     *string    `json:"email"`
+	Name      *string    `json:"name"`
+	State     *string    `json:"state"`
+	Blocked   *bool      `json:"blocked"`
+	CreatedAt *time.Time `json:"created_at"`
```

Comment:
Let's not introduce this pattern right now, so please do not use pointers here...